src/main/target/NERO/target.h

```diff
@@ -136,5 +136,4 @@
 #define TARGET_IO_PORTD         (BIT(2))
 
 #define USABLE_TIMER_CHANNEL_COUNT 9
-
-#define USE_TIMER_MGMT
+#define USED_TIMERS             (TIM_N(3) | TIM_N(5) | TIM_N(8))
```

Comment:
This is actually going the opposite way of where Betaflight is going for 4.0 - eventually most or all of the remaining targets will have `USED_TIMERS` removed, and `USE_TIMER_MGMT` defined, so `timer_definition_unittest.cc` should be reworked to work for targets without `USED_TIMERS` instead.